scraper/src/config/browser_handler.py

```diff
@@ -26,12 +29,42 @@ def init(config_original_content, js_render, user_agent):
             chrome_options.add_argument('--headless')
             chrome_options.add_argument('user-agent={0}'.format(user_agent))
 
-            CHROMEDRIVER_PATH = os.environ.get('CHROMEDRIVER_PATH',
-                                               "/usr/bin/chromedriver")
-            if not os.path.isfile(CHROMEDRIVER_PATH):
-                raise Exception(
-                    "Env CHROMEDRIVER_PATH='{}' is not a path to a file".format(
-                        CHROMEDRIVER_PATH))
+            CHROMEDRIVER_PATH = os.environ.get('CHROMEDRIVER_PATH', '')
+            if not CHROMEDRIVER_PATH or not os.path.isfile(CHROMEDRIVER_PATH):
+                print("Could not find ChromeDriver.")
+                print("Either the Env CHROMEDRIVER_PATH='{}' path is incorrect or "
+                      "ChromeDriver is not installed.".format(CHROMEDRIVER_PATH))
+                print("Do you want to automatically download ChromeDriver?")
+                while(True):
+                    user_input = input("[Y/n]: ")
```

Comment:
I would use only `"[y/n]: "` because the uppercased Y means for Linux users: press enter and the default choice will be "Yes".

Like this:
![image](https://user-images.githubusercontent.com/4116980/153282290-d8726cd4-326c-4389-81aa-d4714470860c.png)